docs/bugs_and_glitches.md

```diff
@@ -2439,7 +2439,7 @@ This bug can prevent you from talking to Eusine in Celadon City or encountering
 ```diff
  	; If there are no monsters in the party,
  	; the player must not own any yet.
--; BUG: CheckOwnMon does not check the Day-Care (see docs/bugs_and_glitches.md)
```

Comment:
Just remove everything in the diff prior to the actual `BUG:` comment.

Reply:
Is replacing that with ... OK too?